demonstrations/tutorial_adaptive_circuits.py

```diff
@@ -214,12 +218,18 @@ def circuit_1(params, excitations):
 # the updated parameters for the selected gates. We also need to define an optimizer. Note that the
 # optimization is not very costly as we only have six gates in our circuit.
 
-opt = qml.GradientDescentOptimizer(stepsize=0.5)
+import optax

```

Comment:
Maybe we could do something like this to give users a bit more clarity:
```suggestion
from optax import sgd as StochasticGradientDescent
```

Reply:
For this I was also following what [this demo](https://pennylane.ai/qml/demos/tutorial_how_to_quantum_just_in_time_compile_vqe_catalyst/) did. But I think your suggestion makes things clearer to the reader.